ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/vllm/426__model_serving_vllm_metrics.robot

```diff
@@ -52,10 +52,24 @@ ${TEST_NS}=                   vllm-gpt2
 ...                           vllm:request_success_total
 ...                           vllm:avg_prompt_throughput_toks_per_s
 ...                           vllm:avg_generation_throughput_toks_per_s
-
+...                           tgi_tokenize_request_tokens_bucket
+...                           tgi_tokenize_request_tokens_count
+...                           tgi_tokenize_request_tokens_sum
+...                           tgi_tokenize_request_input_count_total
+...                           tgi_request_input_count_total
+...                           tgi_request_queue_duration_bucket
+...                           tgi_request_queue_duration_sum
+...                           tgi_queue_size
+...                           tgi_batch_current_size
+...                           tgi_request_input_length_bucket
+...                           tgi_request_input_length_count
+...                           tgi_request_input_length_sum
+...                           tgi_request_generated_tokens_bucket
+...                           tgi_request_generated_tokens_count
+...                           tgi_request_generated_tokens_sum
 
 *** Test Cases ***
-Verify User Can Deploy A Model With Vllm Via CLI
+Verify User Can Deploy A Model With Vllm And tgi Via CLI
     [Documentation]    Deploy a model (gpt2) using the vllm runtime and confirm that it's running
     [Tags]    Tier1    Sanity    Resources-GPU    RHOAIENG-6264   VLLM
     ${rc}    ${out}=    Run And Return Rc And Output    oc apply -f ${DL_POD_FILEPATH}
```

Comment:
This test is not using tgi to deploy the model, it's using vLLM, please revert the name of this TC